src/plugins/table/table-designer-menu.js

```diff
@@ -35,78 +35,98 @@ export const menuItemInfos = {
       ];
       break;
     case DESIGNER_CELL_TYPE.content:
       items = [
-        createMenuItem({
+        createMenuActionItem({
           action: cell.cellType === CELL_TYPE.body ? DESIGNER_CELL_ACTION.convertToHeaderCell : DESIGNER_CELL_ACTION.convertToBodyCell,
           separator: true
         }),
-        createMenuItem({ action: DESIGNER_CELL_ACTION.insertRowBefore }),
-        createMenuItem({ action: DESIGNER_CELL_ACTION.insertRowAfter }),
-        createMenuItem({ action: DESIGNER_CELL_ACTION.deleteRow, disabled: !canDeleteRow, separator: true }),
-        createMenuItem({ action: DESIGNER_CELL_ACTION.insertColumnBefore }),
-        createMenuItem({ action: DESIGNER_CELL_ACTION.insertColumnAfter }),
-        createMenuItem({ action: DESIGNER_CELL_ACTION.deleteColumn, disabled: !canDeleteColumn, separator: true }),
-        createMenuItem({ action: DESIGNER_CELL_ACTION.connectToRowBefore, disabled: cell.isFirstInColumn }),
-        createMenuItem({ action: DESIGNER_CELL_ACTION.connectToRowAfter, disabled: cell.isLastInColumn }),
-        createMenuItem({ action: DESIGNER_CELL_ACTION.connectToColumnBefore, disabled: cell.isFirstInRow }),
-        createMenuItem({ action: DESIGNER_CELL_ACTION.connectToColumnAfter, disabled: cell.isLastInRow }),
-        createMenuItem({ action: DESIGNER_CELL_ACTION.disconnectCell, disabled: !cell.isConnected })
+        createMenuActionItem({ action: DESIGNER_CELL_ACTION.insertRowBefore }),
+        createMenuActionItem({ action: DESIGNER_CELL_ACTION.insertRowAfter }),
+        createMenuActionItem({ action: DESIGNER_CELL_ACTION.deleteRow, disabled: !canDeleteRow }),
+        createMenuSeparatorItem(),
+        createMenuActionItem({ action: DESIGNER_CELL_ACTION.insertColumnBefore }),
+        createMenuActionItem({ action: DESIGNER_CELL_ACTION.insertColumnAfter }),
+        createMenuActionItem({ action: DESIGNER_CELL_ACTION.deleteColumn, disabled: !canDeleteColumn }),
+        createMenuSeparatorItem(),
+        createMenuActionItem({ action: DESIGNER_CELL_ACTION.connectToRowBefore, disabled: cell.isFirstInColumn }),
+        createMenuActionItem({ action: DESIGNER_CELL_ACTION.connectToRowAfter, disabled: cell.isLastInColumn }),
+        createMenuActionItem({ action: DESIGNER_CELL_ACTION.connectToColumnBefore, disabled: cell.isFirstInRow }),
+        createMenuActionItem({ action: DESIGNER_CELL_ACTION.connectToColumnAfter, disabled: cell.isLastInRow }),
+        createMenuActionItem({ action: DESIGNER_CELL_ACTION.disconnectCell, disabled: !cell.isConnected })
       ];
       break;
     default:
       items = [];
       break;
   }
 
+  const menu = <Menu items={items} onClick={handleMenuItemClick} />;
+
   return (
-    <LinkPopover items={items} placement={placement} trigger="click" renderSeparator="onlyWhenSpecified" onVisibleChange={setIsPopoverVisible}>
+    <Dropdown overlay={menu} trigger={['click']} onVisibleChange={handleDropdownVisibleChange} arrow>
```

Comment:
Seems you could set `arrow={{ pointAtCenter: true }}` and the arrow of the dropdown would be right on the dot in the table menus:
<img width="214" alt="Screen Shot 2022-06-24 at 16 12 56" src="https://user-images.githubusercontent.com/8189923/175553820-66078ad3-6495-41be-a27b-c0d9d28d39de.png">
👇 
<img width="213" alt="Screen Shot 2022-06-24 at 16 09 36" src="https://user-images.githubusercontent.com/8189923/175553522-d68bfd46-319f-4acf-8942-07dc0ad7b3eb.png">
👌

Reply:
Wow, done, thanks. I didn't know about this setting. I wonder why anyone would ever want a different default behavior :thinking:

Reply:
Possibly for some cases when the layout cannot afford the dropdown going out of that particular alignment bound of the parent or when it needs to be aligned with the parent 🤔 🤷‍♀️